src/lib.rs

```diff
@@ -62,10 +62,12 @@ pub fn panic_hook(panic_info: &PanicInfo) {
     };
 
     let location = panic_info.location().map(|l| l.to_string());
+    let backtrace = cfg!(feature = "capture-backtrace").then(Backtrace::force_capture);
```

Comment:
Capturing should respect the user preference. We want to use `Backtrace::capture`, not `Backtrace::force_capture`.

Reply:
Resolved in https://github.com/LukeMathWalker/tracing-panic/pull/4/commits/2fb18d08f8edd0d5eae4e69cd51937515ba7beb0

Additionally I left a note to the user explaining how to enable a backtrace if they want one. This matches the behavior of the rust default panic hook.